bokehjs/src/lib/models/filters/index_filter.ts

```diff
@@ -29,7 +29,7 @@ export class IndexFilter extends Filter {
   }
 
   compute_indices(_source: DataSource): number[] | null {
-    if (this.indices != null && this.indices.length >= 0) {
+    if (this.indices != null && this.indices.length > 0) {
```

Comment:
So this change to a strict comparision breaks things for zero length arrays (which is a corner case, but should function). I think the second condition should just be removed altogether.

Reply:
@bryevdv I did on my latest commit, but the js-test check is still failing. Locally I ran all the tests and they passed.